packages/netlify-cms-core/src/components/Editor/EditorControlPane/EditorControl.js

```diff
@@ -328,7 +329,23 @@ class EditorControl extends React.Component {
             />
             {fieldHint && (
               <ControlHint active={isSelected || this.state.styleActive} error={hasErrors}>
-                {fieldHint}
+                <ReactMarkdown
+                  allowedElements={['a', 'strong', 'em']}
```

Comment:
`em` should be used to support italic.

Also, with `unwrapDisallowed=true` we don't need the `p` element or the added styling.